docs/12_0_0/core/contentsecuritypolicy.md

```diff
@@ -34,6 +34,20 @@ Browsers will report CSP violations to a user-defined HTTP endpoint using `repor
 </context-param>
 ```
 
+When using Primefaces in a Portlet context you might want to generate the Nonce and set the CSP header outside of 
+Primefaces. Using `policyProvided` will make the code CSP complient while not setting the CSP Header.
+
+```xml
+<context-param>
+    <param-name>primefaces.CSP</param-name>
+    <param-value>policyProvided</param-value>
+</context-param>
+<context-param>
+    <param-name>primefaces.CSP_REPORT_ONLY_POLICY</param-name>
+    <param-value>report-uri /my-csp-reports</param-value>
+</context-param>
```

Comment:
Can you remove this CSP_REPORT_ONLY_POLICY here in your doc as you don't need it right for your `policyProvided` type right?

Reply:
Yes, this was a copy-paste mistake, sorry.